tests/unit/customizations/ec2/__init__.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
If I had to guess, I think it was before the unit/functional test split where the existing tests were under unit/ but later recategorized as functional.